portfolio/src/main/webapp/script.js

```diff
@@ -102,3 +102,23 @@ $(document).ready(() => {
     fadeDiv('.project');
   });
 });
+
+/**
+ * Adds comments to the page.
+ */
+function addComments() {
+  fetch('/data').then(response => response.json()).then((messages) => {
+    console.log("This is a test.");
```

Comment:
did you mean to leave this console log in here?

Reply:
I didn't, thank you -- I saw and fixed on a later draft, but I missed it here. Fixed :)